Makefile

```diff
@@ -158,11 +159,15 @@ smokecheck: clean develop
 	./scripts/avocado run passtest.py
 
 check: clean develop check_cyclical modules_boundaries
-	selftests/checkall
+	AVOCADO_CHECK_LEVEL=0 selftests/checkall
```

Comment:
One more thing, do you think people would frequently want to run different levels? I'd simply keep `make check` with no handling and `make check-full` which would force `AVOCADO_CHECK_LEVEL=2` (or 99) as those targets probably cover most scenarios....